kapacitor/vars.go

```diff
@@ -197,6 +198,13 @@ func groupBy(q *chronograf.QueryConfig) string {
 	return "[" + strings.Join(groups, ",") + "]"
 }
 
+func idVar(q *chronograf.QueryConfig) string {
+	if len(q.GroupBy.Tags) > 0 {
+		return `name + ':{{.Group}}'`
+	}
+	return "name"
```

Comment:
What was the motivation for this? This change could be potentially problematic, why doesn't the previous way of

```
name + ':{{.Group}}'
```

work?

Reply:
hmm... looks like @alexpaxton changed the base branch from master to feature/persistent-legend.

I think persistent-legend is behind and so A LOT more files ended up in this PR.